connectivity/connectivity-demos/config/rest-clients.yaml

```diff
@@ -9,6 +9,7 @@ RestClients:
     Properties:
       username: theWorker
       password: ${decrypt:\u007B\u0012\u0012\u0075\u002F\u0017\u00D1\u0083\u00C4\u009C\u0094\u00B0\u009A\u00B2\u00ED\u006B\u0067\u0094\u00D2\u003B\u00BF\u0022\u00F3\u00EC\u00C1\u0014\u00A8\u00D8\u00CB\u009C\u0028\u00F3\u00CB\u0030\u00A7\u0038\u00F0\u0067\u0022\u00C3\u0076\u0016\u0088\u00BC\u0096\u00C9\u00EC\u00BB\u002C\u006E\u005E\u008B\u004A\u00AA\u0082\u0057\u00EF\u000A\u0089\u00E1\u0080\u004E\u0072\u0065}
+      JSON.Serialization.WRITE_ENUMS_USING_INDEX: 'true'
```

Comment:
works for me 
![Selection_077](https://github.com/user-attachments/assets/8d431db6-6ed5-4b5f-810d-ad64e333ee18)